### ISSUE TITLE
Able to make Voting Power show incorrectly

### ISSUE THREAD

**Issue by tobias-g1** [julienbh/steemit#1]
#### Expected behavior
The voting power shown against a user on their profile pages should remain correct throughout the experience. 

#### Actual behavior
The voting power shown against a user doesn't remain correct throughout the experience, when you got onto a 100% VP and back to your profile it copies across the voting power. 

Note: It also copies the rep. 

#### How to reproduce
1. Download from here: https://chrome.google.com/webstore/detail/steemit-is-fun/pnpajjflaaacinkffplgcjmloacmogbj
2. Go to Steemit.com
3. Go to your profile page and take note of the voting power (VP) shown against your name (Your voting power should not be 100% for this test).
4. After going on your profile and noting your VP you should go to the profile of a member who has 100% voting power. 
5. Go onto their page and view their profile. Take note of their VP and rep. 
6. When on their profile, click your profile image to bring up the menu and click on blog, this will redirect you to your blog page. You notice that your voting power is now displaying the same as theirs.

Browser: Google Chrome (Version 63.0.3239.132 (Official Build) (64-bit))
Operating system: MacOS HighSierra Version 10.13
Application Version: 0.2.6.2
Updated: January 31, 2018

#### Recording Of The Bug

A recording of the bug of the bug can be found here: https://d.tube/#!/v/tobias-g/aeug2qku

    

<br /><hr/><em>Posted on <a href="https://utopian.io/utopian-io/@tobias-g/able-to-make-voting-power-show-incorrectly">Utopian.io -  Rewarding Open Source Contributors</a></em><hr/>

**Comment by julienbh** [comment]
Yes I also reproduce on back and forward buttons. Just not sure how to handle this yet. 
Thanks for your report. 👍
